airflow-core/pyproject.toml

```diff
@@ -187,7 +187,7 @@ dependencies = [
     "statsd>=3.3.0",
 ]
 "all" = [
-    "apache-airflow-core[graphviz,kerberos,otel,statsd]"
+    "apache-airflow-core[async,graphviz,kerberos,memray,otel,statsd]"
```

Comment:
This change is unrelated? Why do we need it?

Reply:
Not really, no. It demands for methodtools, you can add it like how we do for maybe `timezones`: https://github.com/apache/airflow/blob/main/shared/timezones/pyproject.toml#L28

Reply:
You are right. Fixed.